packages/applications-service-api/__tests__/unit/services/document.v3.service.test.js

```diff
@@ -43,15 +45,89 @@ describe('documentV3 service', () => {
 				rows: DB_DOCUMENTS
 			});
 		});
+
+		it('calls query api with filters if they are specified', async () => {
+			await fetchDocuments({
+				caseReference: 'EN010085',
+				page: 1,
+				filters: [
+					{
+						name: 'category',
+						value: "Developer's Application",
+						type: [{ value: 'Plans' }, { value: 'Reports' }]
+					},
+					{
+						name: 'stage',
+						value: 1,
+						type: [{ value: 'Deadline 2' }, { value: 'Deadline 3' }]
+					}
+				]
+			});
+
+			expect(mockFindAndCountAll).toBeCalledWith({
+				limit: 20,
+				offset: 0,
+				order: [['date_published', 'DESC']],
+				where: {
+					[Op.and]: [
+						{ case_reference: 'EN010085' },
+						{
+							[Op.or]: [
+								{ category: "Developer's Application", filter_1: ['Plans', 'Reports'] },
+								{ stage: 1, filter_1: ['Deadline 2', 'Deadline 3'] }
+							]
+						}
+					]
+				}
+			});
+		});
+
+		it('calls query api with searchTerm if it is specified', async () => {
```

Comment:
Tests look good, we could add a test for searchTerm combined and a Filter, but this will be picked up when we test API.

Reply:
Yep that's a good scenario, I'll add a test for that